source/Octopus.Client/Repositories/CommunityActionTemplateRepository.cs

```diff
@@ -22,7 +22,7 @@ public CommunityActionTemplateRepository(IOctopusRepository repository) : base(r
         public void Install(CommunityActionTemplateResource resource)
         {
             var baseLink = resource.Links["Installation"];
-            var link = Repository.Scope.Apply(spaceId => baseLink.AppendSpaceId(spaceId),
+            var link = Repository.Scope.Apply(space => baseLink.AppendSpaceId(space.Id),
```

Comment:
Would be nice if `.AppendSpaceId` was more strongly typed now. I.e. can we change the signature of `AppendSpaceId` to: `public static string AppendSpaceId(this Href link, SpaceResource space)`

Reply:
done